source/support.tex

```diff
@@ -619,6 +619,7 @@
   // \libheader{deque}, \libheader{queue}, \libheader{stack}, \libheader{string}
 #define @\defnlibxname{cpp_lib_copyable_function}@                 202306L // also in \libheader{functional}
 #define @\defnlibxname{cpp_lib_coroutine}@                         201902L // also in \libheader{coroutine}
+#define @\defnlibxname{cpp_lib_debugging}@                         202606L // freestanding, also in \libheader{debugging}
```

Comment:
Is this, maybe, part of the "emails with suggested editorial changes"?
If so, I'd like to know the reason, as this is surprising.
```suggestion
#define @\defnlibxname{cpp_lib_debugging}@                         202311L // freestanding, also in \libheader{debugging}
```

Reply:
Always, the value of the macro is the date of adoption.
I haven't read the paper, so I don't know if "we're merging this now, but adopting in 2026-06".

Reply:
Typically, the paper's instruction says to set the value to the date of adoption.
@tkoeppe This might be worth adding to the editor's notes.

Reply:
Yes, absolutely: feature-test macros get the year/month of the plenary that voted them into the working draft, not some hypothetical future date when a future standard may or may not be approved (at what stage?)

Please fix as suggested.

Reply:
@JohelEGP Papers don't get to choose the values of those macros, so I don't think we'll call this out.